docling_core/types/doc/document.py

```diff
@@ -5543,27 +5557,61 @@ def get_visualization(
         self,
         show_label: bool = True,
         show_branch_numbering: bool = False,
+        visualizer: str = "reading_order",
```

Comment:
- let's call this something like `viz_mode` (as "visualizer" typically refers to the object) and
- perhaps make it an enum or `Literal` that reflects the options